packages/wallets/src/features/cashier/screens/DepositErrorScreen/DepositErrorContent.tsx

```diff
@@ -0,0 +1,35 @@
+import { TSocketError } from '@deriv/api-v2/types';
+
+type TDepositErrorContentProps = {
+    cryptoConnectionError: boolean;
+    currency?: string;
+    error: TSocketError<'cashier'>['error'];
+    suspendedCurrencyDeposit: boolean;
+};
+
+const getDepositErrorContent = ({
+    cryptoConnectionError,
```

Comment:
Do we need to pass`cryptoConnectionError` and `suspendedCurrencyDeposit` as props? Can't we get it from error.code?

Reply:
Updated and checked for error.code from the content component instead